argon/argon/src/main.cpp

```diff
@@ -30,7 +30,7 @@ std::uint32_t listDevices()
 
 std::uint32_t listChannels()
 {
-    std::uint32_t channels;
+    uint32 channels;
```

Comment:
why?

Reply:
Because of typedef inside ftdi library messing with standard types